.github/ci/vcvarsall.ps1

```diff
@@ -12,7 +12,7 @@
 
 $erroractionpreference = "stop"
 
-cmd /c "`"$env:VCVARSALL`" $VCVARSPLATFORM -vcvars_ver=$VCVARSVERSION & set" |
+cmd /c """$env:VCVARSALL"" $env:VCVARSPLATFORM -vcvars_ver=$env:VCVARSVERSION && set" |
```

Comment:
Are you sure this quoting is correct? I don't know the details of how powershell does quotation grouping, but I would expect a sequence of two quotes `""` to be interpreted as an empty string and be replaced with nothing. Removing the two instances of `""`, you are left with a single level of quotations that quotes the whole thing.

```suggestion
cmd /c "$env:VCVARSALL $env:VCVARSPLATFORM -vcvars_ver=$env:VCVARSVERSION && set" |
```

If that is what you meant, write that. Otherwise, the quotes may need to be changed.

Reply:
This is one of the powershell oddities. Inside a `""` string you can use a `""` to signify a `"`. https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_quoting_rules?view=powershell-7.5

Reply:
And we need to quote VCVARSALL since it is a filepath.

Reply:
Oh, OK. Weird.